hydroshare/storage.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib.staticfiles.storage import ManifestStaticFilesStorage
+
+
+class ForgivingManifestStaticFilesStorage(ManifestStaticFilesStorage):
```

Comment:
I created this class because on my dev rig, running `collectstatic` failed due to some of our 3rd party js libraries importing files that don't exists. Presumably, we run collectstatic during every deploy and it hasn't been failing so likely this isn't necessary?

Reply:
```
Post-processing 'css/jquery-ui/black-tie/jquery-ui.css' failed!

Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.6/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 199, in handle
    collected = self.collect()
  File "/usr/local/lib/python3.6/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 145, in collect
    raise processed
  File "/usr/local/lib/python3.6/site-packages/django/contrib/staticfiles/storage.py", line 303, in _post_process
    content = pattern.sub(converter, content)
  File "/usr/local/lib/python3.6/site-packages/django/contrib/staticfiles/storage.py", line 204, in converter
    force=True, hashed_files=hashed_files,
  File "/usr/local/lib/python3.6/site-packages/django/contrib/staticfiles/storage.py", line 141, in _url
    hashed_name = hashed_name_func(*args)
  File "/usr/local/lib/python3.6/site-packages/django/contrib/staticfiles/storage.py", line 352, in _stored_name
    cache_name = self.clean_name(self.hashed_name(name))
  File "/usr/local/lib/python3.6/site-packages/django/contrib/staticfiles/storage.py", line 101, in hashed_name
    raise ValueError("The file '%s' could not be found with %r." % (filename, self))
ValueError: The file 'css/jquery-ui/black-tie/images/animated-overlay.gif' could not be found with <django.contrib.staticfiles.storage.ManifestStaticFilesStorage object at 0x7fbd678c3828>.
```

Reply:
https://bugs.jqueryui.com/ticket/10134

Reply:
Full log file for collectstatic attempt in case that is helpful:
[collectstatic.TXT](https://github.com/hydroshare/hydroshare/files/8466474/collectstatic.TXT)

Reply:
Can you log out the libraries that fail instead of just moving on?  I'd like to see the list of libraries that fail.  If we log out a warning, we may be able to move this PR forward and update our libraries at a later date.

Reply:
@sblack-usu good call thank you!
added logging in commit bd029355a228e1c52abe34c5612cdcd8e57c7893
Looks like this will go to `hydroshare.log` is that okay or do we need to log elsewhere?

Reply:
added console logging in 9659fef9aad8f94fb4539b290fe3495738a38a8d